package.json

```diff
@@ -256,7 +256,7 @@
     "resin-compose-parse": "^2.1.2",
     "resin-doodles": "^0.1.1",
     "resin-image-fs": "^5.0.9",
-    "resin-multibuild": "^4.7.2",
+    "resin-multibuild": "^4.10.0",
```

Comment:
This one line is the "core feature implementation" really... The rest of the PR is bug fixes (qemu download) and printing hints to the user about the `--pull` and `--emulated` options that become particularly relevant in conjunction with multiarch images.